### PR TITLE
appdata: `translate=no` properties

### PR DESCRIPTION
**Pull request by yakushabb**
It appears that the appstream project no longer supports `translatable=no` properties, and gettext extract the `translatable=no` marked strings as translatable.

I opened an issue to inform about the situation, but `translatable=no` properties are not accepted by developers. You can find the issue here: https://github.com/ximion/appstream/issues/623

**Please test your script or string extraction process before merging this PR.**

> In MetaInfo files, each individual paragraph of a description
> (or enumerated entry) is translated individually, however,
> you can only exclude the complete block from being translated
> by adding `translate="no"` to the description element.

Source: https://freedesktop.org/software/appstream/docs/sect-Quickstart-Translation.html

### CODE REVIEW

**Comment by ryonakano** [comment]
@yakushabb

> gettext extract the `translatable=no` marked strings as translatable.

I'm on Fedora 39 with gettext 0.22 and AppStream 0.16.1 but haven't seen this. Maybe it's changed in AppStream 1.x?

Anyway, I tested the `script or string extraction process` (`ninja com.github.ryonakano.konbucase-pot` command) and these strings won't appear in the generated pot files. Also, `appstreamcli validate` does not claim any errors, so it's looks good to me.

I saw the issue you opened on the appstream repository and found you're suggesting changes to many projects. Thank you very much for your work!

**Review by ryonakano — APPROVED** [review]

File: data/konbucase.metainfo.xml.in [hunk 1]
```diff
@@ -52,26 +52,26 @@
...
   <releases>
     <release version="4.1.2" date="2024-03-02" urgency="low">
-      <description translatable="no">
+      <description translate="no">
         <ul>
           <li>Set minimum window size to prevent it from being cut off</li>
           <li>Send toast when text copied</li>
           <li>Flatpak: Update platform version</li>
-          <li>Metainfo: Mark release descriptions as untranslatable</li>
+          <li>Metainfo: Mark release descriptions as untranslate</li>
```
Comment (ryonakano) [inline data/konbucase.metainfo.xml.in:67]: Ah…maybe you replaced all occurrence or "translatable" in this file :sweat_smile:

```suggestion
          <li>Metainfo: Mark release descriptions as untranslatable</li>
```

**Comment by yakushabb** [comment]
Some gettext version mignt not recognize the "translate" properties and can extract strings. 

Line your case. 

https://github.com/ryonakano/konbucase/commit/33c409d88fc7a729b3c797e39ce5f9e82898d52d

**Comment by ryonakano** [comment]
Yeah…I found that after merging :weary:

**Pull request merged**